cockatrice/src/deckview.cpp

```diff
@@ -10,7 +10,7 @@
 #include <QGraphicsSceneMouseEvent>
 #include <QMouseEvent>
 #include <algorithm>
-#include <math.h>
+#include <cmath>
```

Comment:
um, where are we using cmath instead of qt? that might have portability concerns